experimental/live-editor/src/lib/monaco/schema.js

```diff
@@ -112,12 +112,56 @@ export const schema = {
                                         "type": "string"
                                     },
                                     "name": {
-                                        "description": "Name of the template to apply",
+                                        "description": "Name of the template to apply Only one of Name and Options can be specified.",
                                         "minLength": 1,
                                         "type": "string"
+                                    },
+                                    "options": {
+                                        "description": "Options is a list of template names and Selectors. The templates must all be of type Kind. A template will be selected if the workload matches the specified Selector. Only one template can be selected. Only one of Name and Options can be specified. Minimum number of items in list is two.",
```

Comment:
is the lack of updates to this something that CI could detect like we do for `config/crds`? ala `make generate ; test -z "$(git status --porcelain)"`